tests/xapi-plugins/test_lsblk.py

```diff
@@ -0,0 +1,2 @@
+def test_list_block_devices(host):
```

Comment:
The requirements should specify that you need the lsblk plugin. For convenience, you could check that the plugin exists and skip if it doesn't, as we haven't released it yet.